src/components/Feeds/FeedTags/SocialTags.jsx

```diff
@@ -1,14 +1,31 @@
 import React from "react";
 import { AllTags, Tag, TagsContainer } from "./FeedTagsElements";
 
-const SocialTags = ({ tags, handleClick }) => {
+const SocialTags = ({ tags, handleClick, searchTerm, selectedTags, setSelectedTags }) => {
+    const handleTagSelection = (tag) => {
+        if (selectedTags.includes(tag) || searchTerm.includes(tag)) {
+            setSelectedTags(selectedTags.filter((item) => item !== tag));
+        } else {
+            setSelectedTags([...selectedTags, tag]);
+        }
+        handleClick(tag);
+    };
     return (
         <TagsContainer>
             <AllTags>
                 {tags.slice(0, 10).map(
                     (tag, key) =>
                         tag.length !== 0 && (
-                            <Tag key={key} onClick={() => handleClick(tag)}>
+                            <Tag
+                                style={{
```

Comment:
Why didn't you add those styles inside the 'Tag' styled-component?

Reply:
i thought maybe since an override i put it inline . I should place in the styled component ?

Reply:
https://styled-components.com/docs/basics#passed-props